app/controllers/dashboard/base_controller.rb

```diff
@@ -67,4 +66,8 @@ def establish_breadcrumber
       session[:breadcrumbs] = Dashboard::Breadcrumber.new
     end
   end
+
```

Comment:
In general (besides protocol index), admin is defined by having admin orgs on a protocol